examples/getting-started-app/README.md

```diff
@@ -20,17 +20,46 @@ Features:
 Before you start developing with `cartridge`, you need to install several utilities:
 
 * `git` &mdash; version control system (see details [here](https://git-scm.com/))
-* `npm` &mdash; package manager for `node.js` (see details [here](https://www.npmjs.com/))
 * `cmake` version 2.8 or higher
 * `tarantool-devel` &mdash; developer package for `tarantool`
 * `gcc` &mdash; `C` compiler (see details [here](https://gcc.gnu.org/))
 * `unzip`
 
-To create your project in a quick and easy manner, install the `cartridge-cli`
-tool. Say this:
+To create your project in a quick and easy manner, install the latest tagged version of `cartridge-cli`
+tool (not `scm-1` version).
 
+To see available versions of cartrdge-cli, say:
 ```bash
-you@yourmachine $ tarantoolctl rocks install cartridge-cli
+you@yourmachine $ tarantoolctl rocks search cartridge-cli
+```
+
+You should see something like that:
+```
+cartridge-cli - Search results for Lua 5.1:
+===========================================
+
+
+Rockspecs and source rocks:
+---------------------------
+
+cartridge-cli
+   scm-1 (rockspec) - http://rocks.tarantool.org
+   1.7.0-1 (rockspec) - http://rocks.tarantool.org
+   1.6.0-1 (rockspec) - http://rocks.tarantool.org
+   1.5.0-1 (rockspec) - http://rocks.tarantool.org
+   ...
+```
+
+The latest tagged version of `cartridge-cli` at the moment of writing this article is `1.7.0-1`.
+So in this case you need to install `1.7.0-1` version. Just say this:
+
+```bash
+you@yourmachine $ tarantoolctl rocks install cartridge-cli 1.7.0-1
```

Comment:
Maybe `tarantoolctl rocks install cartridge-cli 1.7.0`? 

Thanks a lot for describing this part!

Reply:
Glad to help